rom/dev/src/main.rs

```diff
@@ -75,13 +75,10 @@ pub extern "C" fn rom_entry() -> ! {
     };
 
     if !cfg!(feature = "no-cfi") {
-        cprintln!("[state] CFI Enabled");
```

Comment:
Some of these are potentially useful to ensure we're running with the right hardware and ROM.